docs/datatable_tutorial/live_stream.md

```diff
@@ -34,7 +34,7 @@ darkTheme = {
 
 Lastly let's add in an API so we can live stream data into our datatable.
 
-Here we use a [Background Task](https://reflex.dev/docs/events/background_events/) to stream the data into the table without blocking UI interactivity. We call an advice API using `httpx` and then append that data to the `self.table_data` state var. We also create a button that allows us to start and pause the streaming of the data by changing the value of the boolean state var `running` using the event handler `toggle_pause`. If the `running` state var is set to `True` we stream the API data, when it is set to `False` we break out of the `while` loop and end the background event.
+Here we use a [Background Task](https://reflex.dev/docs/events/background-events/) to stream the data into the table without blocking UI interactivity. We call an advice API using `httpx` and then append that data to the `self.table_data` state var. We also create a button that allows us to start and pause the streaming of the data by changing the value of the boolean state var `running` using the event handler `toggle_pause`. If the `running` state var is set to `True` we stream the API data, when it is set to `False` we break out of the `while` loop and end the background event.
```

Comment:
We should convert this to a path instead of hardcoding

Reply:
changed